pages/download/index.html

```diff
@@ -0,0 +1,93 @@
+<head>
+    <script src="/resources/scripts/htmlimporter.js"></script>
+    <script>
+        HTMLImporter.import("/resources/templates/generic-head.html");
+    </script>
+    <link rel="stylesheet" href="/resources/stylesheets/download.css" />
+</head>
+
+<body>
+    <script>HTMLImporter.import("/resources/templates/generic-nav.html");</script>
+
+    <div id="content">
+        <h1>Latest Updates on GitHub - Lime3DS</h1>
+        <div id="release" class="release"></div>
+        <div class="navigation">
+            <button id="prev" onclick="navigate(-1)" disabled>Next</button>
+            <button id="next" onclick="navigate(1)" disabled>Previous</button>
+        </div>
+        
+        <script>HTMLImporter.import("/resources/templates/generic-footer.html");</script>
+    </div>
+
+    <script>
+        let releases = [];
+        let currentIndex = 0;
+        document.addEventListener('DOMContentLoaded', () => {
+            fetchReleases();
+        });
+        async function fetchReleases() {
+            try {
+                const response = await fetch("https://api.github.com/repos/Lime3DS/Lime3DS/releases");
+                releases = await response.json();
+                if (releases.length) {
+                    showRelease(currentIndex);
+                    updateNavigationButtons();
+                }
+            } catch (error) {
+                console.error("Error fetching releases:", error);
+            }
+        }
+
+        function showRelease(index) {
+            const releaseDiv = document.getElementById("release");
+            const release = releases[index];
+            const classMap = {
+                windows: "windows",
+                linux: "linux",
+                mac: "mac",
+                android: "android"
+            };
+
+            const assetList = release.assets
+                .map(asset => {
+                    const className = Object.keys(classMap).find(key => asset.name.includes(key));
+                    return className ? `<li><a href="${asset.browser_download_url}" class="button ${classMap[className]}">${asset.name}</a></li>` : '';
+                })
+                .join("");
+
+            releaseDiv.innerHTML = `
+                <h4>Download:</h4>
+                <ul>
+                    ${assetList}
+                    <li>
+                        <p>We are also on Flathub:</p>
```

Comment:
```suggestion
```

Reply:
Stating that we are on Flathub next to the big button that says "Flathub" is redundant

Reply:
> Stating that we are on Flathub next to the big button that says "Flathub" is redundant

i mean... it's the same as on https://github.com/Lime3DS/Lime3DS?tab=readme-ov-file#linux

![image](https://github.com/user-attachments/assets/c66f21bf-aa19-4d09-b052-8bf10cbe69f5)

Reply:
The difference here is that our readme has descriptions for all of our downloads, whereas here the Flathub download is the only one with such text.